keyserver/server.go

```diff
@@ -81,6 +82,8 @@ type ServerPolicies struct {
 // a mechanism to update the underlying ConiksDirectory automatically
 // at regular time intervals.
 type ConiksServer struct {
+	logger zap.Logger
```

Comment:
Related to https://github.com/coniks-sys/coniks-go/pull/146/files#r101622894: I think it'd be nice if we could say `utils.Logger` instead of `zap.Logger`.